app/components/Button/ToggleButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { FunctionComponent } from "react";
+
+import { cn } from "utils";
+
+import { Button } from "./Button";
+
+type TProps = {
+  text: string;
+  activePeriod: string;
+  setActivePeriod: (period: string) => void;
+};
+
+export const ToggleButton: FunctionComponent<TProps> = ({
```

Comment:
This component should be refactored accordingly taking into account that it should be a radio button and a part of the form